packages/plugin-react-performance/lib/with-instrumented-component.tsx

```diff
@@ -30,7 +30,7 @@ class InstrumentedComponent extends React.Component<InstrumentedComponentProps>
   }
 
   async componentDidMount () {
-    if (this.componentMountSpan && BugsnagPerformance.appState === 'ready') {
```

Comment:
@AnastasiiaSvietlova this logic was the wrong way around - the fact we hadn't detected this probably means the unit tests aren't sufficient